homework-submission/my-app/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoItems from './Components/createDynamicItems';
+import Button from './Components/button';
+
+class Todolist extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [
+        {
+          name: 'REACT Lesson',
+          deadline: 23 - 3 - 2019,
+          key: Date.now(),
+        },
+        {
+          name: 'Wach movie',
+          deadline: 23 - 3 - 2019,
+          key: Date.now() + 1,
```

Comment:
`Date.now()` giving a number of how many milliseconds started internet _(my guess or as someones said it's since God created the earth 😅)_.

it happen to be great at that number it giving, and as a unique number to use as an id it's good but I'm sure you can do better generating a unique numbers or strings. you can ask how for a short answer or you may google it for more basic knowledge :)

*Either ways feel free to ask*